pkg/update/update.go

```diff
@@ -66,6 +66,7 @@ const (
   <img src="https://raw.githubusercontent.com/wolfi-dev/.github/b535a42419ce0edb3c144c0edcff55a62b8ec1f8/profile/wolfi-logo-light-mode.svg" />
 </p>
 `
+	updateLabel = "request-version-update"
```

Comment:
Does it make sense to allow the updateLabel to be customized?

Reply:
https://github.com/wolfi-dev/os/pull/1938

Reply:
Updated the code to pass labels as a flag and tested manually: https://github.com/wolfi-dev/os/pull/1938